deltacat/compute/compactor_v2/utils/task_options.py

```diff
@@ -82,15 +87,23 @@ def _get_merge_task_options(
         and round_completion_info.hb_index_to_entry_range
     ):
 
-        previous_inflation = (
-            round_completion_info.compacted_pyarrow_write_result.pyarrow_bytes
-            / round_completion_info.compacted_pyarrow_write_result.file_bytes
+        previous_inflation: float = (
+            (
+                round_completion_info.compacted_pyarrow_write_result.pyarrow_bytes
+                / round_completion_info.compacted_pyarrow_write_result.file_bytes
+            )
+            if round_completion_info.compacted_pyarrow_write_result.file_bytes
```

Comment:
Is `round_completion_info.compacted_pyarrow_write_result` guaranteed to never be None? Should we test this explicitly and raise a more graceful/explicit error if it is None?

Reply:
> Is `round_completion_info.compacted_pyarrow_write_result` guaranteed to never be None?

Yes - the `compacted_pyarrow_write_result` property is non-optional on `RoundCompletionInfo`. See https://github.com/ray-project/deltacat/blob/300b199090caf0d81f54b0a3cfb5318eae4b6bc0/deltacat/compute/compactor/model/round_completion_info.py#L87